README.md

```diff
@@ -13,18 +13,26 @@ JWST Calibration Pipeline
 Installation
 ------------
 
-The `jwst` package can be installed into a virtualenv or conda environment via `pip`.  We recommend creating a fresh environment with only python installed.  If using conda, make sure you have a recent version of Anaconda or Miniconda installed.  Via conda:
+The `jwst` package can be installed into a virtualenv or conda environment via `pip`.  In general, we recommend creating a fresh environment with only python installed.  If using conda, make sure you have a recent version of Anaconda or Miniconda installed.  From a bash shell enter:
 
-    conda create -n jwst_env python
-    conda activate jwst_env
+    conda create -n <env_name> python
+    conda activate <env_name>
+If conda is not in your path, you will need to cd to the bin directory of your anaconda installation and enter:
```

Comment:
Always a bit leery of situations where "its not in the path": For general users, if that is so, then something is probably not right and there will be more issues later on.

Reply:
Agree.  Conda better be in your path.  And you need to fix your conda installation.  That is outside the scope of this document.

Reply:
Well, conda has never been in my path.
I'm running a bog-standard bash shell setup by ITSD. I can't be the only one without conda in their path.